packages/storybook/stories/components/input/input.stories.tsx

```diff
@@ -197,3 +197,23 @@ export const Types: Story = {
     </>
   ),
 };
+
+export const AccessibilityFormField: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <FormField id="input-form-field">
+      <FormFieldLabel htmlFor="form-field-input">
+        Name:
+      </FormFieldLabel>
+
+      <Input id="form-field-input" />
+    </FormField>
+  ),
+};
+
+export const AccessibilityInput: Story = {
+  tags: ['!dev'],
+  render: ({}) => (
+    <Input aria-label='Email' />
```

Comment:
do we actually want to put this as a valid example?
even if it's vocalized, it's bad practice to not put an actual readable label instead of this